lib/solidus_dev_support/templates/extension/spec/spec_helper.rb.tt

```diff
@@ -21,4 +21,6 @@ require '<%= file_name %>/factories'
 RSpec.configure do |config|
   config.infer_spec_type_from_file_location!
   config.use_transactional_fixtures = false
+
+  config.extend Spree::TestingSupport::AuthorizationHelpers::Request, type: :system
```

Comment:
I'm thinking that it may make sense to move this directly inside Solidus, where some [similar code](https://github.com/solidusio/solidus/blob/master/core/lib/spree/testing_support/authorization_helpers.rb#L64) already exists. 

I know in Solidus we don't have (yet) proper `system` specs, we're still using the `feature` denomination, but the moment we do, we would need to remember to remove the code here, as it would be duplicate. WDYT?

Reply:
Good point @spaghetticode, I agree 👍

Reply:
I kinda agree with @spaghetticode. The only gotcha is that if we add it now, it will only be available running specs against Solidus master.

We could fix by doing the PR on Solidus + adding a check runtime here that loads the config only if Solidus < 2.11

Reply:
Draft PR for system specs here: https://github.com/solidusio/solidus/pull/3552